lighthouse-core/scoring.js

```diff
@@ -16,14 +16,14 @@ const clampTo2Decimals = val => Math.round(val * 100) / 100;
 class ReportScoring {
   /**
    * Computes the weighted-average of the score of the list of items.
-   * @param {!Array<{score: number, weight: number|undefined}>} items
+   * @param {Array<{score: number, weight: number}>} items
```

Comment:
turns out we always have score and weight, so letting the type system handle it now